### PR TITLE
feature/4294-Use-Payment-Model

### PR DESCRIPTION
**Pull request by bcdevlucas**
*Issue #:* /bcgov/entity###

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Review by LJTrent — CHANGES_REQUESTED** [review]
remove this from the pr

**Review by LJTrent — CHANGES_REQUESTED** [review]
remove all of the chnage sto new-request-module.ts except the required payment stuff

File: client/src/store/new-request-module.ts [hunk 1]
```diff
@@ -1356,10 +1348,37 @@ export class NewRequestModule extends VuexModule {
...
+      } else if (applicants) {
+        this.setApplicantDetails(applicants)
       }
-      response = await axios.put(`/namerequests/${nrNum}`, data, {
+    } catch (error) {
+      // eslint-disable-next-line
+      console.log(error)
+    }
+  }
+  @Action
```
Comment (LJTrent) [inline client/src/store/new-request-module.ts:1378]: Only add these chnages

File: client/src/store/new-request-module.ts [hunk 2]
```diff
@@ -937,9 +933,7 @@ export class NewRequestModule extends VuexModule {
   get nrNames () {
```
Comment (LJTrent) [inline client/src/store/new-request-module.ts:933]: revert these changes please. I think this is what is causing the check mark to display on the wrong line.

File: client/src/store/new-request-module.ts (hunk 1 above)
Comment (bcdevlucas) [reply client/src/store/new-request-module.ts:1383]: Bad merge, and a couple differences with my auto-format code style cause i'm using webstorm not vscode. Resolved.
